docs/modules/ROOT/pages/data-migration.adoc

```diff
@@ -1,39 +1,64 @@
+* Migrate data to a new cluster before decommissioning your original cluster.
 
-You need at least two clusters with the same configuration. For example, if one cluster
-contains xref:custom-classes-upload.adoc[custom classes] or Persistence,
-you should enable and configure these options for the other cluster as well.
+* Use WAN replication with a xref:failover-clients-with-hazelcast-cloud.adoc[failover client] as part of your disaster recovery strategy. WAN replication automatically synchronizes data between clusters. If a cluster fails, the failover client can switch to an alternative cluster with minimal interruption to service.
+
+== Before you Begin
 
-== Step 1. Create WAN Replication
+* Set up two active clusters with developer mode turned off. Use the same configuration for each cluster. For example, if one cluster
+contains xref:custom-classes-upload.adoc[custom classes] or persistence,
+enable and configure the same options for the other cluster.
 
-In order to create WAN Replication, you can go to *Cluster Detail* > *Settings* > *Wan Replication* and click *Create New Replication*
+* Decide which cluster you want to:
+** Send or migrate data from. This is the _publisher_ cluster.
+** Receive data. This is the _consumer_ cluster.
 
-After setting all these parameters, you can click *Next* to select and configure consumer cluster.
+== Step 1. Set Up WAN Replication
 
-Settings on this screen for consumer cluster are same as publisher cluster except *Active/Passive* section. If you set consumer cluster as *Active*, data will be synchronized from/to publisher and consumer. If you set consumer cluster as *Passive*, data will be sent only from publisher to consumer.
-Once you configured settings for consumer, you can click *Save*
+Set up WAN replication on the publisher cluster to send data to your consumer cluster.
 
-Within seconds, wan replication will be ready there.
+. Go to *Your Clusters* and select your publisher cluster.
+. Select *MANAGE CLUSTER > SETTINGS*.
+. On the *Cluster Settings* page, select *WAN Replication* and then click *+ CREATE NEW REPLICATION*.
++
+The publisher cluster is set as the *Active* cluster. Any data updates performed on this cluster are automatically copied to the other cluster connected by the WAN replication. In this case, your consumer cluster.
++
+. Update the xref:wan-replication.adoc#creating-wan-replication[other parameters] as required.
+. Click *NEXT >* to configure the consumer cluster.
+. From the *Cluster 2* list, select your consumer cluster.
+. From the *Active/Passive* list, select *Passive* to ensure that your consumer cluster only receives data.
++ 
+If you leave the setting as *Active*, any data updates performed on this cluster are automatically copied to your publisher cluster.
++   
+. Click *Save*. WAN Replication is set up.
+. If you want to synchronize all data between the two clusters, click *Sync*. This option is useful if you intend to decommission the publisher cluster.
```

Comment:
Maybe 'shut down' is better than decommission?

Reply:
Updated as suggested.